✨ Add `FanoutAig` that implements `mockturtle::fanout_view` (#116)
marcelwa/aigverse: README.md

    print(f"Level of {node}: {depth_aig.level(node)}")
```

## AIGs with Fanout Information

> You can retrieve the fanouts of a single node from the AIG network, fanout information could be used to calculate factored form literal count(FFLC), which is a good estimate of the hardware cost.

Reviewer: I personally wouldn't restrict the use case in this way. Let the users figure out what they want to use it for.
```suggestion
If needed, you can retrieve the fanouts of AIG nodes as well:
```

Author: Done.

```python
from aigverse import FanoutAig

fanout_aig = FanoutAig(aig)
n_and = aig.get_node(f_and)

> fanout_list = fanout_aig.fanouts(n_and)
> # Iterate over the fanouts of a node
> for fanout in fanout_list:

Reviewer: This can be simplified:
```suggestion
# Iterate over the fanouts of a node
for fanout in fanout_aig.fanouts(n_and):
```

Author: Done.
